lib/__tests__/fixtures/plugin-fixes.mjs

```diff
@@ -0,0 +1,38 @@
+import stylelint from '../../index.mjs';
+
+const {
+	createPlugin,
+	utils: { ruleMessages, validateOptions },
+} = stylelint;
+
+const ruleName = 'plugin/fixes';
+
+const messages = ruleMessages(ruleName, {});
+
+const rule = (primary, _, context) => {
+	return (root, result) => {
+		const validOptions = validateOptions(result, ruleName, {
+			actual: primary,
+			possible: [true],
```

Comment:
off-topic

We have 3 `possible: [true, false],` in the codebase that need to be changed to `possible: [true, null],`.